manifests/p/Piriform/CCleaner/ProTrial/6.13.0.10517/Piriform.CCleaner.ProTrial.installer.yaml

```
@@ -1,29 +1,29 @@
-# Automatically updated by the winget bot at 2023/May/04
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
-
-PackageIdentifier: Piriform.CCleaner.ProTrial
-PackageVersion: 6.11.0.10455
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-Scope: machine
-InstallModes:
-- interactive
-- silent
-UpgradeBehavior: install
-Commands:
-- ccleaner
-FileExtensions:
-- zzz
-Installers:
-- Architecture: x64
-  InstallerUrl: https://download.ccleaner.com/cctrialsetup.exe
-  InstallerSha256: 2E7B593B5620687CF9F85366C5C0BBF2F2DEA132A7C346613E9D64317A77A2F6
-  ProductCode: CCleaner
-- Architecture: x86
-  InstallerUrl: https://download.ccleaner.com/cctrialsetup.exe
-  InstallerSha256: 2E7B593B5620687CF9F85366C5C0BBF2F2DEA132A7C346613E9D64317A77A2F6
-  ProductCode: CCleaner
-ManifestType: installer
-ManifestVersion: 1.2.0
+# Created with YamlCreate.ps1 v2.2.8 $debug=AUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Piriform.CCleaner.ProTrial
+PackageVersion: 6.13.0.10517
```

```suggestion
PackageVersion: 6.13
```
Either need to use Apps&Features DisplayVersion or set PackageVersion to 6.13 to follow the versioning of `Piriform.CCleaner`

---

I was following the same structure as the existing package, where the full version is used. I assume it was done for a reason

---

Well the version used in the ReleaseNotesUrl is `6.13.0.10517` and imo this should've been the versioning scheme used for `Piriform.CCleaner` as well. Looking at the old PRs, it seems you can either blame me or wingetbot as the `FileVersion` is the full 4 part version.

Since we retain only the latest version in the manifest, it doesn't matter much. I'd say we go with adding `DisplayVersion: 6.13` in the manifest and retain the marketing version. This will also mean future versions will be flagged with https://github.com/microsoft/winget-pkgs/labels/Manifest-AppsAndFeaturesVersion-Error in case wingetbot or anyone else messes up the version

---

Sounds like a good approach to me